src/components/NavigationBar/index.js

```diff
@@ -377,6 +380,38 @@ class NavigationBar extends Component {
     });
   };
 
+  togglemenu = () => {
+    this.setState({
+      showMenu: !this.state.showMenu,
+    });
```

Comment:
Pls try implementing the changes using react hooks. We are already trying to switch from the class based components to hooks functionality.

Reply:
I supposed that hooks couldn't be used within classes. Please correct me if i am wrong , I just introduced myself with hooks.

Reply:
Yes, hooks allow to us to play with the states without tye use of classes, they bind the states of the components with a function. 
True, it will take some time to shift to hooks based components from the class based components, that's why it would be better to take the initiative, be it slow.

@akshatnitd pls share your thoughts, should the changes be implemented without react hooks?

Reply:
Yes. But will mixing of hooks and classes be possible in this case ? or are you saying to completely replace classes with hooks?
I am willing to learn and work with React hooks, maybe we can work on it together.
@akshatnitd guide us on this...

Reply:
to use `React Hooks` make the component functional one.

Reply:
@tobehonest cool, I am willing to work on it, too. Let's get to know the decision, first.